packages/node_modules/@ciscospark/react-component-video/src/index.js

```diff
@@ -13,19 +13,29 @@ export default function Video({src, srcObject, audioMuted}) {
     }
   }
 
+  if (videoMuted) {
```

Comment:
We probably don't want this feature any more since we are splitting tracks. Otherwise, we'll have two audio tags on the page.

Reply:
Good call